hexa/datasets/models.py

```diff
@@ -363,14 +365,25 @@ def custom_encoding(item):
             elif isinstance(item, list):
                 return [custom_encoding(element) for element in item]
             elif isinstance(item, bytes):
-                return base64.b64encode(item).decode("utf-8")
+                return self._encode_bytes(item)
             return item
 
         # Preprocess the object to replace NaN values with None and encode bytes to base64
         obj = custom_encoding(obj)
         # Use the superclass's encode method to serialize the preprocessed object
         return super().encode(obj)
 
+    def _encode_bytes(self, item):
+        try:
+            return base64.b64encode(item).decode("utf-8")
+        except Exception:
+            try:
+                polygone = wkb.loads(item, hex=True)
+                geo_json = shapely.geometry.mapping(polygone)
+                return geo_json
+            except Exception:
+                return None
```

Comment:
So we fail silently in this case ? Maybe we should fail or at least have a warning, WDYT ?

Reply:
If we assume we won't have byte encoded tabular data then I am on board to discard it. So far it offers the option to preview the parquet file with wkb binary data in it. As in the example for geometry  it will display a dictionary with base64 encoded geometry field.
It does not bring much information to the data scientist working with it except knowing that it is geometry data..

Reply:
Let's assume that.